### ISSUE TITLE
Optimize screen reader accessibility of `Popover` component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2785]
Scope
---
The current implementation of accessibility support can be enhanced by announcing the text contents automatically for screen reader users if the `Popover` component is in open state. 

Actually the contents must be actively activated by using special screen reader commands to reach the text contents.

Notes  
---
- Aria live regions could be a proper way to implement this behavior.
- Slack: https://porschedev.slack.com/archives/C01NLDJJ83Z/p1695135817377449

Acceptance criteria
---
- [ ] If the `Popover` has open state, the text contents should be announced automatically for screen reader users.

Subtasks
---
- [ ] Task

**Comment by o-hook** [comment]
Will be fixed with #4516
